game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/commands/CommandFactory.scala

```diff
@@ -62,6 +67,39 @@ class CommandFactory {
         return new UseBoostCommand;
     }
 
-    private def makeUseOilCommand(): RawCommand = new UseOilCommand
+    private def makeUseOilCommand(): RawCommand = {
+        return new UseOilCommand
+    }
```

Comment:
Does this function differently than `private def makeUseOilCommand(): RawCommand = new UseOilCommand` ?

Reply:
Probably not, can make it shorter 👍